packages/quantic/package.json

```diff
@@ -56,7 +56,7 @@
     "@octokit/graphql-schema": "10.74.2",
     "@salesforce/eslint-config-lwc": "3.3.3",
     "@salesforce/eslint-plugin-lightning": "1.0.0",
-    "@salesforce/sfdx-lwc-jest": "1.1.2",
+    "@salesforce/sfdx-lwc-jest": "1.3.0",
```

Comment:
This change undoes some overrides from the root `package.json` which are meant to fix Quantic unit tests.

If unit tests pass despite this, then I recommend removing the overrides from the root `package.json` for `sfdx-lwc-jest`.

Reply:
all right change has been reverted.